samples/browser/src/main/java/org/mozilla/samples/browser/addons/AddonsFragment.kt

```diff
@@ -131,9 +136,13 @@ class AddonsFragment : Fragment(), AddonsManagerAdapterDelegate {
         requireContext().components.addonManager.installAddon(
             addon,
             onSuccess = {
+                val text = getString(
```

Comment:
I feel like we should just inline getString() inside of Toast.makeText instead of having a variable.

Reply:
Yeah, I think so too, let's inline @Amejia481 .

So, basically:

```
Toast.makeText(
  requireContext(),
  getString(R.string.mozac_feature_addons_successfully_installed, it. translatedName)
  Toast.LENGTH_SHORT
).show()
```

Reply:
Let's do that `ktlint` is happy is with :)

Reply:
`ktlint` is such a complainer... :)

Reply:
:)